examples/full/clickpipe/kafka_azure_eventhub/main.tf

```diff
@@ -0,0 +1,74 @@
+variable "service_id" {
+  description = "ClickHouse service ID"
+}
+
+variable "kafka_brokers" {
+  description = "Kafka brokers"
+}
+
+variable "kafka_topics" {
+  description = "Kafka topics"
+}
+
+variable "azure_eventhub_connection_string" {
+  description = "Connection string for Azure EventHub"
+  sensitive   = true
+}
+
+resource "clickhouse_clickpipe" "kafka_azure_eventhub" {
+  name        = "Azure EventHub 🚀 ClickPipe"
+  description = "Data pipeline from Azure EventHub to ClickHouse"
+
+  service_id = var.service_id
```

Comment:
examples in the `examples/full` directory are meant to be self sufficient, meaning people should be able to just set the variables for authentication and `terraform apply` the example. In this case, the service is not created in the example.
Please note examples in `examples/full` are used by e2e tests and thus this test would never pass.